### ISSUE TITLE
FlatPak version?

### ISSUE THREAD

**Issue by mands** [klauscfhq/tusk#79]
## Description

Would be great to have a FlatPak version similar to the Snap package that could then be installed from GNOME Software via https://flathub.org/home

## Relevant Links (if any)

- https://flatpak.org/
- https://flathub.org/home

**Comment by Thomas-Howard** [comment]
I would also like to see this, Flatpak provides a tidier experience for me when using Gnome with Wayland on Fedora with a high DPI display and I trust it more than I trust Snap.

**Comment by muldengold** [comment]
Yes, a Flatpak version would be most welcome! Thanks a lot for the great job so far!!!!!

**Comment by tbenst** [comment]
This would be awesome. AppImage has worse portability than flatpak (I just failed to get the AppImage running on NixOS)

**Comment by Red-Eyed** [comment]
Any update?
